### PR TITLE
Fix split view on iOS

### PR DESCRIPTION
**Pull request by yogevbd**

### CODE REVIEW

File: lib/ios/UIViewController+LayoutProtocol.m [hunk 1]
```diff
@@ -60,6 +60,14 @@ - (UIInterfaceOrientationMask)supportedInterfaceOrientations {
 	return interfaceOrientationMask;
 }
 
+- (UINavigationController *)stack {
+    if ([self isKindOfClass:UINavigationController.class]) {
+        return (UINavigationController *)self;
+    } else {
+        return self.navigationController;
```
Comment (guyca) [inline lib/ios/UIViewController+LayoutProtocol.m]: I think this should be `self.parentViewController.stack`.

**Comment by exentrich** [comment]
@yogevbd Huge thanks for this fix! You rocks!🤘Now split view on iPad works as expected!

**Comment by guyca** [comment]
@exentrich I'm sorry but I didn't quite understand your question. Could you please push a repo with an example of the scenario in question? thanks.

**Comment by exentrich** [comment]
@guyca @yogevbd I forked RNN and created additional screens for SplitView in Playground app:
https://github.com/exentrich/react-native-navigation

Please check this detailed slides to fully understand my issue:
https://docs.google.com/presentation/d/e/2PACX-1vROJdzLbLvLF0h8gk-udRVxbxuQAqcMxFkQEeWhMZVwsyhD6tEAB8zLxyThFKinNHFDvLOxVVa9W38o/pub?start=false&loop=false&delayms=10000

I'm really interested to start using SplitViewLayout, and I can share any more details as you want!

**Pull request merged**